src/utils/display.py

```diff
@@ -150,7 +153,7 @@ def display_video_motpy(
     resize_shape=None,
     scale=None,
     model="Mediapipe",
-    extract_face=False,
+    extract_face=True,
```

Comment:
also change this also to false, i guess, as reverting main.py and not this could cause ambiguities.